src/test/kotlin/note/mothers/NoteMother.kt

```diff
@@ -28,4 +29,10 @@ object NoteMother {
     }
 
     fun getIdentifierFrom(note: Note) = note.id
+
+    fun getValidNoteFromIdentifier(identifier: Identifier) = Note(
```

Comment:
Isn't really needed, check the comment I told you inside the NotificationCreatorTest file.